eoscompanion/main.py

```diff
@@ -157,6 +157,31 @@ def do_activate(self):  # pylint: disable=arguments-differ
         logging.info('Activated')
         return Gio.Application.do_activate(self)
 
+CONFIG_FILE = '/etc/eos-companion-app/config.ini'
```

Comment:
This path should not be hardcodeded. Have a look at https://github.com/endlessm/eos-companion-app-integration/blob/master/configuration-manager/main.c#L64 for details on which configuration file to load.